<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SelectedImageDetailsWidget</class>
 <widget class="QWidget" name="SelectedImageDetailsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>250</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Selected Image Details</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayoutMain">
   <property name="spacing">
    <number>6</number>
   </property>
   <property name="leftMargin">
    <number>6</number>
   </property>
   <property name="topMargin">
    <number>6</number>
   </property>
   <property name="rightMargin">
    <number>6</number>
   </property>
   <property name="bottomMargin">
    <number>6</number>
   </property>
   <item>
    <widget class="QGroupBox" name="groupBoxImageInfo">
     <property name="title">
      <string>画像情報</string>
     </property>
     <layout class="QGridLayout" name="gridLayoutImageInfo">
      <property name="spacing">
       <number>3</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="labelFileName">
        <property name="text">
         <string>ファイル名:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="labelFileSize">
        <property name="text">
         <string>ファイルサイズ:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="labelFileNameValue">
        <property name="text">
         <string>-</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLabel" name="labelFileSizeValue">
        <property name="text">
         <string>-</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="labelCreatedDate">
        <property name="text">
         <string>登録日:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLabel" name="labelCreatedDateValue">
        <property name="text">
         <string>-</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="labelImageSizeValue">
        <property name="text">
         <string>-</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="labelImageSize">
        <property name="text">
         <string>解像度:</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBoxAnnotationSummary">
     <property name="title">
      <string>アノテーション概要</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayoutAnnotationSummary">
      <item>
       <widget class="QGroupBox" name="groupBoxTags">
        <property name="title">
         <string>タグ</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayoutTags">
         <item>
          <widget class="QLabel" name="labelTagsContent">
           <property name="text">
            <string>cat, sitting, outdoor, wooden</string>
           </property>
           <property name="wordWrap">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBoxCaptions">
        <property name="title">
         <string>キャプション</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayoutCaptions">
         <item>
          <widget class="QTextEdit" name="textEditCaptionsContent">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>1</verstretch>
            </sizepolicy>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
           <property name="placeholderText">
            <string>キャプションが表示されます</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayoutRatingScore">
        <item row="0" column="0">
         <widget class="QLabel" name="labelRating">
          <property name="text">
           <string>Rating:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLabel" name="labelRatingValue">
          <property name="text">
           <string>PG-13</string>
          </property>
          <property name="visible">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QComboBox" name="comboBoxRating">
          <property name="visible">
           <bool>true</bool>
          </property>
          <item>
           <property name="text">
            <string>PG</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>PG-13</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>R</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>X</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>XXX</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QPushButton" name="pushButtonSaveRating">
          <property name="text">
           <string>保存</string>
          </property>
          <property name="visible">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="labelScore">
          <property name="text">
           <string>スコア:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLabel" name="labelScoreValue">
          <property name="text">
           <string>75.4</string>
          </property>
          <property name="visible">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QSlider" name="sliderScore">
          <property name="minimum">
           <number>0</number>
          </property>
          <property name="maximum">
           <number>1000</number>
          </property>
          <property name="value">
           <number>500</number>
          </property>
          <property name="orientation">
           <enum>Qt::Orientation::Horizontal</enum>
          </property>
          <property name="visible">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QPushButton" name="pushButtonSaveScore">
          <property name="text">
           <string>保存</string>
          </property>
          <property name="visible">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="AnnotationDataDisplayWidget" name="annotationDataDisplay" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>1</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>AnnotationDataDisplayWidget</class>
   <extends>QWidget</extends>
   <header>..widgets.annotation_data_display_widget</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
